### PR TITLE
Remove page_size

### PR DESCRIPTION
**Pull request by pjuftring**
Removes every trace of existence of page_size from the source, the
testfiles and README.md
https://github.com/WebAssembly/design/pull/442

### CODE REVIEW

File: ml-proto/host/params.ml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-let page_size = 4096L
+let page_size = 0x10000L (* 64 KiB *)
```
Comment (rossberg-chromium) [inline ml-proto/host/params.ml]: Given that this is no longer a parameter, can you instead remove this definition completely? It should also vanish from the `host_params` type in the eval module.

**Comment by sunfishcode** [comment]
Keeping page_size as a named variable improves the "specyness" of the code, and makes it easier to find all the places in the spec that case about page sizes. I would prefer to leave it in.

**Comment by rossberg-chromium** [comment]
Fair enough. The only remaining occurrence is in the evaluation of GrowMemory. I suggest exporting page_size as a constant from the memory module and using that there.

**Comment by sunfishcode** [comment]
lgtm

**Comment by rossberg-chromium** [comment]
LGTM

**Pull request merged**